packages/mendel-development-middleware/index.js

```diff
@@ -39,11 +39,55 @@ function MendelMiddleware(opts) {
         return acc;
     }, {});
 
+    var allDirs = existingVariations.reduce(function(allDirs, variation) {
+        variation.chain.forEach(function(path) {
+            if (-1 === allDirs.indexOf(path)) allDirs.push(path);
+        });
+        return allDirs;
+    }, []);
+
+    console.log(allDirs);
+
     var loader = new MendelLoader(existingVariations, config, module.parent);
 
     return function(req, res, next) {
         req.mendel = req.mendel || {};
 
+        req.mendel.getBundleEntries = function() {
+            console.log(bundles)
```

Comment:
I was under the impression you would continue this work and I left it there to help understanding the current state things are working.